src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/LicenceService.kt

```diff
@@ -1,6 +1,7 @@
 package uk.gov.justice.digital.hmpps.hmppshdcapi.licences
 
 import com.fasterxml.jackson.databind.ObjectMapper
+import jakarta.persistence.EntityNotFoundException
```

Comment:
**issue:** think this is the wrong type of exception - it's JPA one and not mapped in the controller advice. 
We might need to add a custom exception (worth what we do in CVL) and it'd be worth adding an integration test to ensure its being marshalled into the correct error response